scripts/common.php

```diff
@@ -103,6 +103,20 @@ function get_com_en_name($sci_name) {
   return $engname;
 }
 
+function get_sci_name($com_name) {
+  if (!isset($_labels_flickr)) {
+    $_labels_flickr = file(get_home() . "/BirdNET-Pi/model/labels.txt");
```

Comment:
re-using `_labels_flickr` will break get_com_en_name() since one needs `labels_flickr.txt` and the other `labels.txt`

Reply:
Thanks ; bad copy paste